src/app/core/entity/schema/entity-schema-field.ts

```diff
@@ -66,6 +67,14 @@ export interface EntitySchemaField {
    */
   additional?: any;
 
+  /**
+   * (Optional) Assign any custom "extension" configuration you need for a specific datatype extension.
+   *
+   * You can pass any kind of value here to allow complex custom datytypes' transformations
+   * that are not part of the core datatypes and therefore not included in this core interface.
+   */
+  entityReferenceRole?: EntityReferenceRole;
```

Comment:
I am very confused by the code comment. Doesn't this just allow to set `composite` or `aggregate`? What are these transformations/extensions?

Reply:
thanks! :see_no_evil:   this seems to be a copy of the code comment for `additional` only

Reply:
```suggestion
  /**
   * (Optional) If the dataType of this field references another entity,
   *    define the role of this relationship for the entity containing this field.
   * 
   * i.e. how "important" is the entity this field is referencing?
   * Does this the entity containing this field (not the referenced entity) still have meaning after the referenced entity has been deleted?
   * 
   * see options of the `EntityReferenceRole` type
   */
  entityReferenceRole?: EntityReferenceRole;
```